src/win32.ts

```diff
@@ -74,30 +45,26 @@ export class WindowsApp extends App {
     return this.normalizeIconExtension('.ico');
   }
 
-  needsRcedit() {
-    return Boolean(this.opts.icon || this.opts.win32metadata || this.opts.appCopyright || this.opts.appVersion || this.opts.buildVersion);
+  needsResedit() {
+    return Boolean(this.opts.icon || this.opts.win32metadata || this.opts.appCopyright || this.opts.appVersion || this.opts.buildVersion || this.opts.name);
```

Comment:
Looks like this was broken before, but I believe this might _technically_ be a breaking change in the following scenario:

- the user-provided Packager config does not specify any of the options we're checking for here
- the app's `package.json` does not specify the `version` and `author` fields

In that case, Packager is currently not editing the .exe metadata, and with this change it would use the `name` field from the Packager options (or from the apps's `package.json` `name` / `productName` fields) for the file description, product name and original filename fields. That makes sense, but should we be concerned about someone relying on these fields being just `Electron` / `electron.exe` even if they're passing different values or nah?

Sample `package.json`:

```json
{
  "name": "resedit-test-app",
  "//productName": "Product Name from packageJson",
  "main": "src/index.js",
  "scripts": {
    "package": "electron-packager . --overwrite"
  },
  "devDependencies": {
    "@electron/packager": "file:C:\\path\\to\\local-electron-packager",
    "electron": "29.1.5"
  }
}
```